src/api/Metadata.js

```diff
@@ -352,6 +353,96 @@ class Metadata extends File {
         return editors;
     }
 
+    /**
+     * Utility to concat instance and template into one entity.
+     *
+     * @param {Object} instance - metadata instance
+     * @param {Object} template - metadata template
+     * @return {Object} metadata template instance
+     */
+    createTemplateInstance(instance: MetadataInstanceV2, template: MetadataTemplate): MetadataTemplateInstance {
+        const metadataFields: MetadataInstanceTemplateField[] = [];
+
+        if (template.templateKey !== METADATA_TEMPLATE_PROPERTIES) {
+            // Get Metadata Fields for Instances created from predefinied template
+            const templateFields = template.fields || [];
+            templateFields.map(async field => {
```

Comment:
forEach should be more appropriate here since we don't care about the returned array value. .map may cause some confusion

Reply:
done!